src/v2/mod.rs

```diff
@@ -53,7 +53,8 @@ pub mod manifest;
 mod tags;
 pub use self::tags::StreamTags;
 
-mod blobs;
+pub mod blobs;
```

Comment:
This exposes the whole module in the public API. It is not required here, and in general it is not something you want to do as it is easily results in exposing unintended internal details.